discoverer/src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ Copyright 2014 SeaClouds
+ Contact: SeaClouds
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ 
+ -->
+<web-app>
+
+  <display-name>SeaClouds Discoverer WS Layer</display-name>
+  <description>trololo</description>
+
+
+  <!-- ADD OFFERING -->
+  <servlet>
+	<servlet-name>discoverer_add_offering</servlet-name>
+	<servlet-class>eu.seaclouds.platform.discoverer.ws.AddOffer</servlet-class>
+  </servlet>
+
+  <servlet-mapping>
+	<servlet-name>discoverer_add_offering</servlet-name>
+	<url-pattern>/addoffer</url-pattern>
+  </servlet-mapping>
+  
```

Comment:
@buccarel I see you created several servlets but you're only exposing `/addoffer`. Is this intentional?